docs/mpaso.rst

```diff
@@ -0,0 +1,156 @@
+          type="output"
+          io_type="pnetcdf"
+          filename_template="mpaso.hist.am.layerVolumeWeightedAverage.$Y-$M-$D.nc"
+          filename_interval="00-01-00_00:00:00"
+          output_interval="none"
+          clobber_mode="truncate"
+          packages="layerVolumeWeightedAverageAMPKG">
+
+      <var name="xtime"/>
+      <var name="daysSinceStartOfSim"/>
+      <var_array name="minValueWithinOceanLayerRegion"/>
+      <var_array name="maxValueWithinOceanLayerRegion"/>
+      <var_array name="avgValueWithinOceanLayerRegion"/>
+      <var_array name="minValueWithinOceanVolumeRegion"/>
+      <var_array name="maxValueWithinOceanVolumeRegion"/>
+      <var_array name="avgValueWithinOceanVolumeRegion"/>
+  </stream>
+
+  <stream name="meridionalHeatTransportOutput"
+          type="output"
+          io_type="pnetcdf"
+          filename_template="mpaso.hist.am.meridionalHeatTransport.$Y-$M-$D.nc"
+          filename_interval="00-01-00_00:00:00"
+          output_interval="none"
+          clobber_mode="truncate"
+          packages="meridionalHeatTransportAMPKG">
+
+      <var name="xtime"/>
+      <var name="daysSinceStartOfSim"/>
+      <var name="binBoundaryMerHeatTrans"/>
+      <var name="meridionalHeatTransportLatZ"/>
+      <var name="meridionalHeatTransportLat"/>
+      <var name="refZMid"/>
+      <var name="refBottomDepth"/>
+  </stream>
+
+  <stream name="surfaceAreaWeightedAveragesOutput"
+          type="output"
+          io_type="netcdf"
+          filename_template="mpaso.hist.am.surfaceAreaWeightedAverages.$Y-$M-$D.nc"
+          filename_interval="00-01-00_00:00:00"
+          output_interval="none"
+          clobber_mode="truncate"
+          packages="surfaceAreaWeightedAveragesAMPKG">
+
+      <var name="xtime"/>
+      <var name="daysSinceStartOfSim"/>
+      <var_array name="minValueWithinOceanRegion"/>
+      <var_array name="maxValueWithinOceanRegion"/>
+      <var_array name="avgValueWithinOceanRegion"/>
+  </stream>
+
+  <stream name="mixedLayerDepthsOutput"
+          type="output"
+          io_type="pnetcdf"
+          filename_template="mpaso.hist.am.mixedLayerDepths.$Y-$M-$D.nc"
+          filename_interval="00-01-00_00:00:00"
+          output_interval="none"
+          clobber_mode="truncate"
+          packages="mixedLayerDepthsAMPKG">
+
+      <var name="xtime"/>
+      <var name="daysSinceStartOfSim"/>
+      <stream name="mesh"/>
+      <var name="tThreshMLD"/>
+      <var name="dThreshMLD"/>
+      <var name="tGradMLD"/>
+      <var name="dGradMLD"/>
+  </stream>
+
+The ``filename_tempalate`` can be modified as desired (in most cases, these are
+the defalult values from E3SM).  For the ``timeSeriesStatsMonthlyOutput``
+stream, both the filename_interval and the output_interval must currently be
+monthly (``"0000-01-00_00:00:00"``).
+
+Additional fields can be included in the ``timeSeriesStatsMonthlyOutput``
+streams.  These are the minimum that allow the analysis to run successfully.
```

Comment:
I'm assuming this is from the wiki we used in this process.  Thanks for including it here!

Reply:
These will become pages in the official documentation on ReadTheDocs as soon as this is merged.